lib/amazonka/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## Unreleased
+
+- `amazonka`: During DEBUG logging, do not print bytestrings that have unprintable characters [\#952](https://github.com/brendanhay/amazonka/pull/952)
```

Comment:
Please put the PR link on the next line to match existing entries. I'd also remove the word "DEBUG", since we're fixing this by changing typeclasses.

Reply:
done.